src/lib/rrconfig.js

```diff
@@ -0,0 +1,60 @@
+/*
+  * Licensed Materials - Property of IBM
+  * (C) Copyright IBM Corp. 2016. All Rights Reserved.
+  * US Government Users Restricted Rights - Use, duplication or
+  * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+  */
+'use strict';
+const env = require('./env');
+const DBManager = require('./dbManager');
+const classesView = 'classes/byClass';
+const rrDb = new DBManager({localDbName: 'rr', remoteDbName: env.cloudantDb});
+
+const RRCONFIG_KEY = Symbol.for('hubot-ibmcloud-cognitive-lib-rrconfig');
+
+let globalSymbols = Object.getOwnPropertySymbols(global);
+if (globalSymbols.indexOf(RRCONFIG_KEY) < 0) {
+	global[RRCONFIG_KEY] = {};
+}
+
+let singleton = {};
+Object.defineProperty(singleton, 'instance', {
+	get: function(){
+		return global[RRCONFIG_KEY];
+	}
+});
```

Comment:
The code above is specific to our NLC implementation.  I don't think this implementation is using the singleton created here. Pleaser remove lines 13-25 if it isn't being used.